doc/source_names.tex

```diff
@@ -28,33 +28,35 @@ \section*{Names}
 $\texttt{\textbf{=>}}$ (and \texttt{\textbf{let}} in Source \S3 and 4), the following
 names refer to primitive functions and constants:
 \begin{itemize}
-\item \lstinline{math_}$\textit{name}$,
+\item \href{https://www.comp.nus.edu.sg/~cs1101s/sicp/chapters/6\#footnote-link-4}{\lstinline{math_}$\textit{name}$},
```

Comment:
https://www.comp.nus.edu.sg/~cs1101s/sicp/chapters/6#footnote-link-4 works for me, but not when copying the link from the source because of the escaped `\#`. When the link is clicked in the pdf, it works.

Reply:
https://www.comp.nus.edu.sg/~cs1101s/sicp/chapters/6#footnote-link-3

I mean if you click this, it will jump to the correct reference [3], but there is no [4] to jump to.